dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/provider/ProviderFactory.java

```diff
@@ -796,6 +773,13 @@ private void addBusExtension(List<Object> extensions, Class<?>... extClasses) {
 
     @SuppressWarnings("unchecked")
     protected void setCommonProviders(List<ProviderInfo<? extends Object>> theProviders) {
+        //Liberty code change start
+        List<ProviderInfo<MessageBodyReader<?>>> newReaders = new ArrayList<ProviderInfo<MessageBodyReader<?>>>();
+        List<ProviderInfo<MessageBodyWriter<?>>> newWriters = new ArrayList<ProviderInfo<MessageBodyWriter<?>>>();
+        List<ProviderInfo<ContextResolver<?>>> newResolvers = new ArrayList<ProviderInfo<ContextResolver<?>>>();
+        List<ProviderInfo<ContextProvider<?>>> newProviders = new ArrayList<ProviderInfo<ContextProvider<?>>>();
```

Comment:
sort of a nit, but could we rename this variable from `newProviders` to `newContextProviders`?  technically all of these lists contain JAX-RS providers, and I think `newProviders` could cause confusion while maintaining this code.